releases/latest/beta/Dockerfile.ubuntu.adoptopenjdk11.bak

```diff
@@ -1,7 +1,7 @@
 FROM adoptopenjdk:11-jre-openj9
 ARG LIBERTY_VERSION=21.0.0.9-beta
```

Comment:
This should be 21.0.0.10-beta

Also, the file name shouldn't have `.bak` at the end